modules/claims_api/lib/bd/bd.rb

```diff
@@ -74,13 +100,14 @@ def upload(claim:, pdf_path:, pctpnt_vet_id: nil, doc_type: 'L122', file_number:
     def generate_upload_body(claim:, doc_type:, pdf_path:, pctpnt_vet_id: nil, file_number: nil, original_filename: nil)
       payload = {}
       auth_headers = claim.auth_headers
-      veteran_name = "#{auth_headers['va_eauth_firstName']}_#{auth_headers['va_eauth_lastName']}"
+      veteran_name = compact_veteran_name(auth_headers['va_eauth_firstName'], auth_headers['va_eauth_lastName'])
       birls_file_num = auth_headers['va_eauth_birlsfilenumber'] || file_number if doc_type != 'L705'
-      claim_id = doc_type == 'L705' ? claim.claim_id : claim.evss_id
+      claim_id = get_claim_id(doc_type, claim)
       file_name = generate_file_name(doc_type:, veteran_name:, claim_id:, original_filename:)
-      participant_id = pctpnt_vet_id if doc_type == 'L705'
+      participant_id = pctpnt_vet_id if %w[L075 L190 L750].include?(doc_type)
+      system_name = 'LH-B' if %w[L075 L190].include?(doc_type)
```

Comment:
I believe we're on hold for this ticket as 'LH-B' was not a potential for submitters. We need to either a.) choose from the list returned or b.) make a decision of which of the available values we'll use. 

awaiting return of @klwatanabe

Reply:
@FonzMP, what do you mean by 'LH-B' was not a potential for submitters? I thought we were sending 'LH-B' today for VBMS uploads. Are you saying that the Documents API won't accept that?

Reply:
Oh, sorry, missed the comment on the ticket where @tycol7 noted the returned error from Benefits Documents API.
`Invalid content Source 'LH-B'. Expected one of VBMS, Lighthouse... `

We should double check with team Firefly to confirm that we should use 'Lighthouse' vs 'LH-B'. The goal is to be able to track that the document was updated by Lighthouse. Our team may have been the only ones using 'LH-B' if 'Lighthouse' is available now. 

If we do go that route, we should confirm how it is displayed in VBMS UAT for those uploads.

Reply:
Per Slack, go with `Lighthouse` for now (to test and re-evaluate if needed).

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/17587/commits/8a1c09075b51c3a19d56b4e2c19ab6f4255f1b66